teps/0091-trusted-resources.md

```diff
@@ -180,15 +181,16 @@ Task would still be considered verified.
 **Author publishing a Resource.**
 A Resource Author may want their Resource and its
 outputs to be trusted so that provenance attestations can be confidently made. The author will use their own private key to sign the Resource and signatures are stored in `Annotations` map of the Resource.
-The private keys are generated, used and stored by the author. Public keys then are stored as secrets or via URI stored in the configmap. Similar mechanism is used in [Chains](https://github.com/tektoncd/chains).
+The private keys are generated, used and stored by the author. Public keys then are stored as secrets or via URI stored in the configmap. Similar mechanism is used in [Chains](https://github.com/tektoncd/chains). Or via other install-level configuration to better support multiple keys.
 
 Situation A: Users want to use their own resources and make sure they are not tampered before using in Tekton. The user will need to choose a certain key tool (In the scope of this TEP we will leverage [sigstore](https://github.com/sigstore) to support different keys like cosign or KMS keys) and generate private&public key pair. The user will use the signing cli we provided to sign the yaml files and signature is stored in the `Annotations`.
-The public key can be configured from Tekton Pipeline's configmap or in the taskrun/pipelinerun.
+The public key can be configured at Tekton Pipeline.
```

Comment:
Can you clarify what `at Tekton Pipeline` means? As written, this could mean "as part of the installation of the Pipeline CRD" or "at the time of the installation of this particular `Pipeline` resource"?

Reply:
Oh yes, I will clarify this.  It should be "as part of the installation of the Pipeline CRD", like the configmap.